hardware/Netatmo.cpp

```diff
@@ -2090,11 +2090,11 @@ bool CNetatmo::ParseDashboard(const Json::Value& root, const int DevIdx, const i
 		v << ";";
 		v << rain_24;
 		std::string sValue = v.str().c_str();
-		SendRainSensor(ID, batValue, m_RainOffset[ID] + m_OldRainCounter[ID], name, rssiLevel);
+		SendRainSensor(5, batValue, m_RainOffset[ID] + m_OldRainCounter[ID], name, rssiLevel);
 		std::string Name = name + "-updatevalueint";
                 ///Debug(DEBUG_HARDWARE, "(%d) %s (%s) [%s] rain %s %s %d %d", Hardware_int, str_ID.c_str(), pchar_ID, name.c_str(), sValue.c_str(), m_Name.c_str(), rssiLevel, batValue);
 		//UpdateValueInt(0, str_ID.c_str(), 0, pTypeRAIN, sTypeRAINByRate, rssiLevel, batValue, '0', v.str().c_str(), Name, 0, m_Name);
-		SendRainSensor(ID, batValue, rain_1 + rain_24, name + "- rain", rssiLevel);
+		SendRainSensor(ID, batValue, rain_24 + rain_1, name + "- rain", rssiLevel);
```

Comment:
Why do you use the ID and not a fixed ID of 5 as above (for the same device)?

Note: This is a device ID, not a type...